pygaggle/run/evaluate_passage_ranker.py

```diff
@@ -77,12 +78,15 @@ def tokenizer_sane(cls, v: str, values, **kwargs):
 
 
 def construct_t5(options: PassageRankingEvaluationOptions) -> Reranker:
-    loader = CachedT5ModelLoader(options.model_name_or_path,
-                                 SETTINGS.cache_dir,
-                                 'ranker',
-                                 options.model_type,
-                                 SETTINGS.flush_cache)
     device = torch.device(options.device)
+
+    try:
```

Comment:
Instead of try/except, should we pass a flag to distinguish PT from TF checkpoints?

Reply:
@rodrigonogueira4 

Yes, I feel a flag makes more sense. will add an flag `--from_tf` for `evaluate_passage_ranker.py`

Reply:
@ronakice oh, I didn't notice there is a one change difference between my gcloud and local. Will remove that line.

Reply:
Huggingface's work-around is something like `bool("ckpt" in model_name_or_path)`

Reply:
model_name_or_path is a directory may contains both "ckpt" and `pytorch_model.bin` ? https://huggingface.co/google/bert_uncased_L-10_H-128_A-2#list-files

Reply:
To avoid problems with PT files containing `ckpt` in their name, I would just go with the flag.